spec/order_summary_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'order_summary'
+
+describe OrderSummary do
+
+  subject(:order_summary) { described_class.new }
+  # let(:order) { double('order') }
+
+  context 'order summary' do
+
+    it 'contains a log of the order' do
+      # allow(order).to receive(:add_item).with("Golden French Fries") { {"Golden French Fries" => 1} }
+      # order.add_item("Golden French Fries")
+      expect(order_summary.basket).to eq({})
+    end
+  end
+
+  # context 'it verifies the final order' do
+  #   before do
+  #     allow(order).to receive(:total_price) {3}
+  #   end
+  #
+  #     it 'raise an error if an order is invalid' do
+  #       expect {order_summary.confirm_order(2)}.to raise_error "Sorry our records don't match, we believe the total price is £3.  Please check your order."
```

Comment:
Metrics/LineLength: Line is too long. [157/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/AsciiComments: Use only ascii symbols in comments. (https://github.com/bbatsov/ruby-style-guide#english-comments)